outputs/awssecuritylake.go

```diff
@@ -196,54 +198,80 @@ func getAWSSecurityLakeSeverity(priority types.PriorityType) (int32, string) {
 func (c *Client) EnqueueSecurityLake(falcopayload types.FalcoPayload) {
 	offset, err := c.Config.AWS.SecurityLake.Memlog.Write(c.Config.AWS.SecurityLake.Ctx, []byte(falcopayload.String()))
 	if err != nil {
-		if err.Error() != "future offset" {
-			go c.CountMetric(Outputs, 1, []string{"output:awssecuritylake.", "status:error"})
-			c.Stats.AWSSecurityLake.Add(Error, 1)
-			c.PromStats.Outputs.With(map[string]string{"destination": "awssecuritylake.", "status": Error}).Inc()
-			log.Printf("[ERROR] : %v SecurityLake. - %v\n", c.OutputType, err)
-			return
-		}
+		go c.CountMetric(Outputs, 1, []string{"output:awssecuritylake.", "status:error"})
```

Comment:
I remembered I faced `future offset` error a lot of times, this is why I excluded it because for me, it wasn't a real error.

Reply:
Are you sure this was in `Write`? `ErrFutureOffset` is only returned on reads.

Reply:
I'm not sure anymore, you may be right, let close this comment.